scripts/data/loadLandCover.yml

```diff
@@ -29,34 +9,59 @@ inputs:
       - IO
       - PC
     example: "IO"
+  stac_path:
+    label: STAC catalogue path
+    description: path to the STAC catalogue (url)
+    type: text
+    example: "https://io.biodiversite-quebec.ca/stac/"
+  collections:
+    label: collections
+    description: name of the collection
+    type: text
+    example: c("esacci-lc")
```

Comment:
c() is the R notation for a list. Do I understand that this should be a text[] instead? Or is it a single value?
Label is plural but description is singular, so I can't draw a conclusion from the description...

Reply:
You are right, I changed it to: label: collection and example: "esacci-lc"